butter_cms/tests/test_vcr_config.py

```diff
@@ -1,3 +1,8 @@
+import socket
+def guard(*args, **kwargs):
+    raise Exception("Blocked network access. Use VCR cassettes instead.")
+socket.socket = guard
+
```

Comment:
Out of curiosity, would it also block connection to a database/redis/etc if we use one, considering that we're blocking the whole socket?

I know we don't use them here, but I'm curious if similar solution can be used in e.g. a sophisticated Django app or not.

Reply:
@Toreno96 This file will be only executed during tests, I tested it with django starter and moved tests into root outside of main module for possible mistakes in build commands. 👍🏻

Reply:
Yes, it would block all network connections.

Reply:
Ok, thanks.